monasca/templates/role.yaml

```diff
@@ -16,4 +16,17 @@ rules:
       - services
       - componentstatuses
       - storageclasses
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1alpha1
```

Comment:
Just a note, this won't work in kubernetes 1.6 since the RBAC was promoted from alpha -> beta.